addons/stock/migrations/8.0.1.1/post-migration.py

```diff
@@ -652,15 +654,39 @@ def migrate_procurement_order(cr, registry):
 def migrate_stock_qty(cr, registry):
     """Reprocess stock moves in state done to fill stock.quant."""
     stock_move_obj = registry['stock.move']
-
+    stock_picking_obj = registry['stock.picking']
     done_move_ids = stock_move_obj.search(cr, uid, [('state', '=', 'done')])
     openupgrade.message(
         cr, 'stock', 'stock_move', 'state',
         'Reprocess %s stock moves in state done to fill stock.quant',
         len(done_move_ids))
+    # Recreate stock.pack.operation
+    openupgrade.logged_query(
+        cr,
+        """
+        INSERT INTO stock_pack_operation
+        (picking_id, product_id, product_uom_id, product_qty, qty_done,
+         lot_id, date, location_id, location_dest_id, processed)
+        SELECT
+            picking_id, product_id, product_uom, product_uom_qty,
+            product_uom_qty, %s, date, location_id, location_dest_id, 'true'
+        FROM stock_move
+        WHERE
+            id IN %%s
+            AND picking_id IS NOT NULL
```

Comment:
You're right. I think the correct approach is to recreate all quants processing all stock moves ordering by id (or maybe date), ignoring their source. I close this one and make another PR across the day.